### PR TITLE
Icons: Beaker, Flasks, Pipette

### PR DESCRIPTION
**Pull request by Andreto**
**Adds:**
Beaker, Conical flask, Round Flask, Pipette
**Satisfies Requests:**
Beaker: https://github.com/feathericons/feather/issues/752

image
100%
image

### CODE REVIEW

**Comment by locness3** [comment]
Love that pipette.

**Comment by ericfennis** [comment]
@Andreto Awesome work !

File: icons/beaker.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M4.5 3h15"/>
```
Comment (ericfennis) [inline icons/beaker.svg:12]: Can you put a white space before the closing the tag?
```suggestion
  <path d="M4.5 3h15" />
```

Comment (Andreto) [reply icons/beaker.svg:12]: Should i do that to all of them?

Comment (ericfennis) [reply icons/beaker.svg:12]: Yes please

Comment (Andreto) [reply icons/beaker.svg:12]: I realized that the icons from #118 are missing this spacing as well. Could I sneak the changes for these icons into this PR?

Comment (ericfennis) [reply icons/beaker.svg:12]: Hmm, I think it's better to create a new PR for that one that's easier to track the changes

**Review by locness3 — APPROVED** [review]

**Pull request merged**